src/actions/navigation/index.ts

```diff
@@ -7,3 +8,14 @@ export const navigate = (options: NavigationNavigateActionPayload) => {
 export const goBack = () => {
   return NavigationActions.back()
 }
+
+export const navigatorReset = () => {
+  return NavigationActions.reset({
+    index:0,
+    actions: [ 
+      navigate({
+        routeName: routeList.Home 
```

Comment:
Can we take this out? The action is named navigatorReset, no indication that it travels to Home.
There might be a reason to use this action in cases where we don't want to go to Home right away.